plenum/server/replicas.py

```diff
@@ -154,14 +154,13 @@ def all_instances_have_primary(self) -> bool:
         return all(replica.primaryName is not None
                    for replica in self._replicas.values())
 
-    # TODO unit test
     @property
-    def primaries(self) -> dict:
-        return {r.instId: r.name for r in self._replicas.values()}
+    def primary_name_by_inst_id(self) -> dict:
+        return {r.instId: r.primaryName.split(":", maxsplit=1)[0] for r in self._replicas.values()}
```

Comment:
Should we use Primary's Node Name instead of doing the split? Like `r.node.name`?